Airports/EGOS/Positions.txt

```diff
@@ -1,3 +1,3 @@
-EGOS_APP:Shawbury Approach:133.150:OSA:A:EGOS:APP:-:-:0220:0246:N052.47.52.000:W002.40.00.000::
+EGOS_APP:Shawbury Approach:133.150:OSR:A:EGOS:APP:-:-:0220:0246:N052.47.52.000:W002.40.00.000::
```

Comment:
Does it provide a radar service?

Reply:
Again miliatry unit always use approach but gives full services